smithyctl/internal/command/root.go

```diff
@@ -0,0 +1,19 @@
+package command
```

Comment:
This is internal only as it will be used only to put together the OSS smithyctl binary.

Reply:
Let's keep as is for now if you don't mind - if it's problematic we can refactor it in the next PRs :)